exporters/geneva/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(opentelemetry-fluentd)
+
+add_definitions(-DHAVE_CONSOLE_LOG)
+
+find_package(opentelemetry-cpp REQUIRED)
+find_package(CURL REQUIRED)
+
+include_directories(include)
+# create geneva metrics exporter
+add_library(opentelemetry_exporter_geneva_metrics
+            src/exporter.cc src/unix_domain_socket_data_transport.cc)
+target_include_directories(opentelemetry_exporter_geneva_metrics
+                           PRIVATE ${OPENTELEMETRY_CPP_INCLUDE_DIRS})
+target_link_libraries(opentelemetry_exporter_geneva_metrics
+                      PUBLIC ${OPENTELEMETRY_CPP_LIBRARIES})
+set_target_properties(opentelemetry_exporter_geneva_metrics
+                      PROPERTIES EXPORT_NAME metrics)
+
+if(BUILD_TESTING)
+  if(EXISTS ${CMAKE_BINARY_DIR}/lib/libgtest.a)
+    message("Exits")
```

Comment:
nit: make the message more verbose?

Reply:
These were for my debuggig. Have removed them. Thanks for noticing.